lib/marked.js

```diff
@@ -47,39 +66,32 @@ block.list = edit(block.list)
   .replace('def', '\\n+(?=' + block.def.source + ')')
   .getRegex();
 
-block._tag = '(?!(?:'
```

Comment:
Why was this using and underscore before? Was it supposed to be private?

Reply:
Fair question. There were four underscored variables all together. I could not discern a rationale behind it...even if they were meant to be private. It looked like misunderstood convention to be honest.

The only members that need to be set outside the block object are the ones making calls to `edit()` and the like. All the rest can be within the object; however, there were three or four outside of the block object. Each of them had the underscore prefix...none of which appear to be referenced anywhere else.